packages/smooth_app/lib/pages/preferences/account_deletion_webview.dart

```diff
@@ -0,0 +1,66 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:openfoodfacts/utils/OpenFoodAPIConfiguration.dart';
+import 'package:provider/provider.dart';
+import 'package:smooth_app/data_models/user_preferences.dart';
+import 'package:smooth_app/helpers/user_management_helper.dart';
+import 'package:smooth_app/widgets/smooth_scaffold.dart';
+import 'package:webview_flutter/webview_flutter.dart';
+
+class AccountDeletionWebview extends StatefulWidget {
+  @override
+  AccountDeletionWebviewState createState() => AccountDeletionWebviewState();
+}
+
+class AccountDeletionWebviewState extends State<AccountDeletionWebview> {
```

Comment:
This class should be private

Reply:
Yep the english version looks like the only one work, stick to english smarter ^^